velocity/src/main/java/me/leoko/advancedban/velocity/VelocityMethods.java

```diff
@@ -0,0 +1,384 @@
+    }
+    return false;
+  }
+
+  @Override
+  public String parseJSON(InputStreamReader json, String key) {
+    JsonObject element;
+    try {
+      element = JsonParser.parseReader(json).getAsJsonObject();
+    } catch (IllegalStateException e) {
+      return null;
+    }
+    JsonElement obj = element.get(key);
+    return obj != null ? obj.toString().replaceAll("\"", "") : null;
+  }
+
+  @Override
+  public String parseJSON(String json, String key) {
+    JsonObject element;
+    try {
+      element = (JsonObject) JsonParser.parseString(json).getAsJsonObject();
+    } catch (IllegalStateException e) {
+      return null;
+    }
+    JsonElement obj = element.get(key);
+    return obj != null ? obj.toString().replaceAll("\"", "") : null;
+  }
+
+  @Override
+  public Boolean getBoolean(Object file, String path) {
+    return getConfigNode(file, path).getBoolean();
+  }
+
+  @Override
+  public String getString(Object file, String path) {
+    return getConfigNode(file, path).getString();
+  }
+
+  @Override
+  public Long getLong(Object file, String path) {
+    return getConfigNode(file, path).getLong();
+  }
+
+  @Override
+  public Integer getInteger(Object file, String path) {
+    return getConfigNode(file, path).getInt();
+  }
+
+  @Override
+  public List<String> getStringList(Object file, String path) {
+    try {
+      return getConfigNode(file, path).getList(TypeToken.of(String.class));
+    } catch (ObjectMappingException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  @Override
+  public boolean getBoolean(Object file, String path, boolean def) {
+    return getConfigNode(file, path).getBoolean(def);
+  }
+
+  @Override
+  public String getString(Object file, String path, String def) {
+    return getConfigNode(file, path).getString(def);
+  }
+
+  @Override
+  public long getLong(Object file, String path, long def) {
+    return getConfigNode(file, path).getLong(def);
+  }
+
+  @Override
+  public int getInteger(Object file, String path, int def) {
+    return getConfigNode(file, path).getInt(def);
+  }
+
+  @Override
+  public boolean contains(Object file, String path) {
+    return getConfigNode(file, path).isEmpty();
+  }
+
+  @Override
+  public String getFileName(Object file) {
+    return "[Only Available in the Bukkit Version!]";
+  }
+
+  @Override
+  public void callPunishmentEvent(Punishment punishment) {
+    server.getEventManager().fire(new PunishmentEvent(punishment));
+  }
+
+  @Override
+  public void callRevokePunishmentEvent(Punishment punishment, boolean massClear) {
+    server.getEventManager().fire(new RevokePunishmentEvent(punishment, massClear));
+  }
+
+  @Override
+  public boolean isOnlineMode() {
+    return server.getConfiguration().isOnlineMode();
+  }
+
+  @Override
+  public void notify(String perm, List<String> notification) {
+    server.getAllPlayers().forEach(player -> {
+      if(player.hasPermission(perm)) notification.forEach(str -> sendMessage(player, str));});
```

Comment:
I would recommend cleaning this up slightly:
```suggestion
    server.getAllPlayers().forEach(player -> {
        if (player.hasPermission(perm)) {
            notification.forEach(str -> sendMessage(player, str));
        }
    });
```